src/Discovery/aggMDSUtils.jsx

```diff
@@ -104,4 +104,15 @@ const loadStudiesFromAggMDS = async () => {
   return studies;
 };
 
-export default loadStudiesFromAggMDS;
+const loadAuthMappingsFromWTS = async () => {
+  const res = await fetch(`${wtsAggregateUserUserPath}?filters=authz`);
+  // don't throw error when user isn't logged in
+  if (res.status !== 200) {
+    return {};
```

Comment:
Would this breaks determining access premissions for anonymous and/or authenticated users (who are not in user.yaml)?

If the answer is yes, we should fix the logic